agent/util-scripts/pbench-tool-meister-start

```diff
@@ -739,359 +804,408 @@ def main(_prog: str, cli_params: Namespace) -> int:
     redis_server_spec = cli_params.redis_server
     tds_server_spec = cli_params.tool_data_sink
 
-    if cli_params.orchestrate == "create":
-        orchestrate = True
-        ssh_cmd = "ssh"
-        ssh_path = find_executable(ssh_cmd)
-        if ssh_path is None:
-            logger.error("required ssh command not in our PATH")
-            return ReturnCode.MISSINGSSHCMD
-
-        # Connect with each of the tool hosts: if we can't connect via ssh,
-        # this isn't going to work, so if any of them fails we'll terminate
-        # with an error code.
-        #
-        # If we can connect, they'll tell us the IP address from which they see
-        # us connecting, and we'll use that (by default) as the server address.
-        localhost = LocalRemoteHost()
-        origin_ip = set()
-        any_remote = False
-        template = TemplateSsh(ssh_cmd, shlex.split(ssh_opts), "echo ${SSH_CONNECTION}")
-        for host in tool_group.hostnames.keys():
-            if not localhost.is_local(host):
-                any_remote = True
-                template.start(host)
+    recovery = Cleanup(logger)
```

Comment:
FYI: the changes from here on really confuse GitHub's diff engine. Selecting "Hide whitespace" will help it a lot and minimize confusion. 😁